src/main/resources/multipleHearingLocations.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
(Minor) formatting

Reply:
Is the spacing/new line between brackets '{' and the first property a problem here?

Reply:
I find  "Chester":[
"226511",
"443014"
],
easier to read

Reply:
Refactored